File: app/src/main/java/forpdateam/ru/forpda/api/news/models/NewsItem.java
package forpdateam.ru.forpda.api.news.models;

import java.util.ArrayList;

/**
 * Created by radiationx on 28.08.17.
 */

public class NewsItem {
    private int id;
    private String url;
    private String title;
    private String description;
    private String author;
    private String date;
    private String imgUrl;
    private int commentsCount;
    private ArrayList<Tag> tags = new ArrayList<>();

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getImgUrl() {
        return imgUrl;
    }

    public void setImgUrl(String imgUrl) {
        this.imgUrl = imgUrl;
    }

    public int getCommentsCount() {
        return commentsCount;
    }

    public void setCommentsCount(int commentsCount) {
        this.commentsCount = commentsCount;
    }

    public ArrayList<Tag> getTags() {
        return tags;
    }

    public void addTag(Tag tag) {
        this.tags.add(tag);
    }
}

File: app/src/main/java/forpdateam/ru/forpda/entity/remote/news/NewsItem.kt
package forpdateam.ru.forpda.entity.remote.news

import forpdateam.ru.forpda.entity.DeferredData
import forpdateam.ru.forpda.entity.remote.others.user.User
import ru.radiationx.coretypes.ArticleId

/**
 * Created by radiationx on 28.08.17.
 */

data class NewsItem(
    val id: ArticleId,
    val author: User,
    val url: String,
    val title: String,
    val description: String,
    val date: String,
    val imgUrl: String,
    val commentsCount: Int,
    val tags: List<Tag>,
    val avatar: DeferredData<String>?
)
